### PR TITLE
Small fixes 1

### PR DESCRIPTION
**Pull request by revolunet**
minor fixes/cleanup

### CODE REVIEW

File: packages/code-du-travail-frontend/package.json [hunk 1]
```diff
@@ -8,7 +8,8 @@
   "main": "src/index.js",
   "dependencies": {
     "@sentry/browser": "^4.1.0",
-    "@socialgouv/code-du-travail-ui": "^1.2.3",
+    "@socialgouv/code-du-travail-css": "^1.0.2",
+    "@socialgouv/code-du-travail-ui": "^1.1.0",
```
Comment (lionelB) [inline packages/code-du-travail-frontend/package.json]: why not the `^1.2.3` version ?

File: packages/code-du-travail-frontend/src/Categories.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { Link } from "../routes";
 
 import { Container, Categories } from "@socialgouv/code-du-travail-ui";
 
-import themes from "./data/themes2";
+import themes from "./data/themes";
```
Comment (lionelB) [inline packages/code-du-travail-frontend/src/Categories.js:6]: this will probably confilct with my renaming to `json` 
will see

**Review by lionelB — APPROVED** [review]
Remove! Remove! remove!

File: packages/code-du-travail-frontend/package.json (hunk 1 above)
Comment (revolunet) [reply packages/code-du-travail-frontend/package.json]: totally :)

Comment (revolunet) [reply packages/code-du-travail-frontend/package.json]: i will move that into https://github.com/SocialGouv/code-du-travail-numerique/pull/176

**Pull request merged**